servers/visual/portals/portal_occlusion_culler.h

```diff
@@ -260,7 +260,7 @@ class PortalOcclusionCuller {
 
 		Occlusion::PolyPlane poly;
 		uint32_t flags;
-#ifdef TOOLS_ENABLED
+#if defined(DEV_ENABLED) || defined(TOOLS_ENABLED)
```

Comment:
Without this change it's not possible to build `debug` binaries for `iOS` and `tvOS` platforms.